mangle-shebangs.sh

```diff
@@ -3,9 +3,10 @@
 SYSTEM_RELEASE_PATH=/etc/system-release
 RHEL8_REGEX="Red Hat Enterprise Linux release 8"
 FEDORA_REGEX="Fedora release"
+CENTOS8_REGEX="CentOS Linux release 8"
 
 # RHEL8 and Fedora30+ both treat shebangs of the form "#!/usr/bin/env python" as errors
```

Comment:
Can you also modify the comment? 

RHEL8, Fedora30+ and `CentOS8` both treat shebangs of the form "#!/usr/bin/env python" as errors

Reply:
Comment updated